packages/rest/src/rest.server.ts

```diff
@@ -844,3 +848,12 @@ export interface RestServerOptions {
  * @interface RestServerConfig
  */
 export type RestServerConfig = RestServerOptions & HttpServerOptions;
+
+/**
+ * Options for handling static assets
+ */
+export interface StaticAssetsEntry {
```

Comment:
AFAICT, this interface is no longer used. Please remove.

Reply:
Removed.